google/cloud/pubsub/internal/tracing_message_callback_test.cc

```diff
@@ -46,42 +47,85 @@ using ::google::cloud::testing_util::SpanKindIsInternal;
 using ::google::cloud::testing_util::SpanNamed;
 using ::google::cloud::testing_util::SpanWithParent;
 using ::testing::AllOf;
+using ::testing::ByMove;
 using ::testing::Contains;
+using ::testing::Return;
 
 pubsub::Subscription TestSubscription() {
   return pubsub::Subscription("test-project", "test-sub");
 }
 
 std::shared_ptr<MessageCallback> MakeTestMessageCallback(
-    std::shared_ptr<MessageCallback> mock) {
+    std::shared_ptr<MessageCallback> mock, Options opts) {
   return MakeTracingMessageCallback(
       std::move(mock),
-      Options{}.set<pubsub::SubscriptionOption>(TestSubscription()));
+      opts.set<pubsub::SubscriptionOption>(TestSubscription()));
 }
 
 TEST(TracingMessageCallback, UserCallback) {
   namespace sc = opentelemetry::trace::SemanticConventions;
   auto span_catcher = InstallSpanCatcher();
   auto mock = std::make_shared<pubsub_testing::MockMessageCallback>();
   EXPECT_CALL(*mock, user_callback).Times(1);
-  auto message_callback = MakeTestMessageCallback(std::move(mock));
-
+  auto message_callback = MakeTestMessageCallback(std::move(mock), Options{});
+  auto mock_ack_handler =
+      std::make_unique<pubsub_testing::MockExactlyOnceAckHandlerImpl>();
   auto span = internal::MakeSpan("my-sub subscribe");
```

Comment:
super nit: nobody cares, but `s/my-sub/test-sub/` for naming consistency

Reply:
Done